diff --git gettw.diagram/src/gettw/diagram/edit/parts/RectangleEditPart.java gettw.diagram/src/gettw/diagram/edit/parts/RectangleEditPart.java
index 1e66957..b241750 100644
--- gettw.diagram/src/gettw/diagram/edit/parts/RectangleEditPart.java
+++ gettw.diagram/src/gettw/diagram/edit/parts/RectangleEditPart.java
@@ -132,1 +132,1 @@
-			locator.setBorderItemOffset(new Dimension(-20, -20));
+			locator.setBorderItemOffset(new Dimension(-8, -8));